lib-multisrc/libgroup/src/eu/kanade/tachiyomi/multisrc/libgroup/LibGroup.kt

```diff
@@ -567,7 +584,8 @@ abstract class LibGroup(
             entryValues = arrayOf("main", "secondary", "compress")
             summary = "%s \n\nВыбор приоритетного сервера изображений. \n" +
                 "По умолчанию «Первый». \n\n" +
-                "ⓘВыбор другого помогает при долгой автоматической смене/загрузке изображений текущего."
+                // "ⓘВыбор другого помогает при долгой автоматической смене/загрузке изображений текущего."
```

Comment:
remove commented if not needed

Reply:
I changed the wording of that sentence to make it more clear, and was thinking to leave it for future contributors just in case.
But oh well git still has history so deleted commented line